src/main/viewManager.js

```diff
@@ -50,7 +51,7 @@ export class ViewManager {
       if (recycle && recycle.isVisible) {
         setFocus = recycle.name;
       }
-      if (recycle && recycle.server.url === server.url) {
+      if (recycle && urlUtils.parseURL(recycle.server.url).toString() === urlUtils.parseURL(server.url).toString()) {
```

Comment:
This was needed because one of these URLs would have a `/` embedded at the end, so in order to avoid semantics I just reparsed them. Definitely open to a better solution here if needed.

Reply:
maybe we should try and normalize in its original input (when added to the config and when read from the config) so we don't have to care about that in any other place.